src/components/ColumnHeaders.js

```diff
@@ -1,32 +1,32 @@
 import React from 'react';
-import { TableCell, TableHead, TableRow } from 'material-ui/Table';
 import ReactTooltip from 'react-tooltip';
-import TrackedStats from '../lib/TrackedStats.js';
+import TrackedStats from '../lib/TrackedStats.js'
 
 export default () => (
-    <TableHead className="headerStyle">
-      <TableRow>
-        <TableCell>Player Name</TableCell>
+    <thead className="headerStyle">
+      <tr>
+        <th>Player Name</th>
         {
           Object.values(TrackedStats).map((stat, statIndex) => {
             return (
               <ColumnHeader key={statIndex} index={statIndex} stat={stat}/>
             );
           })
         }
-      </TableRow>
-    </TableHead>
+      </tr>
+    </thead>
 )
 
 const ColumnHeader = ({index, stat}) => {
   return (
-    <TableCell 
+    <th 
+      class="mdl-data-table__cell--non-numeric"
```

Comment:
This should be className. Giving me an error. Looks the same with or without.  I'll fix before merging.